src/components/socio-carddesktop.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+const ColaboradorCardDesktop: React.FC = () => {
+  return (
+    <div className="relative flex h-[192px] w-full max-w-[1300px] items-center overflow-hidden rounded-xl bg-blue-600 p-6 text-white">
+      <div className="absolute bottom-0 left-0 h-[87px] w-[148px] md:h-[148px] md:w-[236px]">
+        <img
+          src="/sociocard.svg"
+          alt="Colaborador Icon"
+          className="h-full w-full object-contain"
+        />
+      </div>
+      <div className="ml-[236px] flex-grow">
+        <h4 className="font-orbitron mb-2 text-3xl">Torna-te colaborador</h4>
```

Comment:
The Orbitron font is actually in the `font-title` class, since we use it for titles. This is configured in the `tailwind.config.js` here:

```js
fontFamily: {
  sans: ["var(--font-inter)", ...fontFamily.sans],
  title: ["var(--font-orbitron)", ...fontFamily.mono],
},
```

The `font-sans` one is for the default font used in the body text (Inter) but is already applied by default everywhere since I added that tailwind class in `layout.tsx`.